stdlib/2and3/socket.pyi

```diff
@@ -529,8 +578,9 @@ class socket:
     def connect(self, address: Union[_Address, bytes]) -> None: ...
     def connect_ex(self, address: Union[_Address, bytes]) -> int: ...
     def detach(self) -> int: ...
+    def dup(self) -> socket: ...
     def fileno(self) -> int: ...
-
+    def get_inheritable(self) -> bool: ...
```

Comment:
Only available since Python 3.4 (so not in Python 2). Should have a version check.